docs/notebooks/01_merge.ipynb

```diff
@@ -1,27 +1,505 @@
 {
```

Comment:
<p>I thought this could be some kind of final validation, even though it is a bit repetitive ...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/icbi-lab/atlas_protocol/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>I'm all for this, but here I would actually raise the exception to make it harder to (accidentically) ignore</p><p> </p>

Reply:
<p>if you have columns that are not valid, you can exclude them prior to the final validation. </p>